spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```
@@ -1064,6 +1064,7 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
           require(copiedModuleParamTable.get(name) != None, s"cloned module should have for $name")
           setLayerWeightAndBias(params,
             copiedModuleParamTable.get(name).get.asInstanceOf[Table], deepCopy)
+        case _=>
```

you should throw an exception here, or mismatch will do nothing.